docs/proposals/authentication-filter.md

```diff
@@ -906,7 +829,98 @@ This can use the status `RouteConditionPartiallyInvalid` defined in the Gateway
 ## Testing
 
 - Unit tests
-- Functional tests to validate behavioural scenarios when referencing filters in different combinations. The details of these tests are out of scope for this document.
+- Functional tests to validate behavioural scenarios when referencing filters in different combinations.
+
+## Functional Test Cases
+
+### Invalid AuthenticationFilter scenarios
+
+This section covers configuration deployment scenarios for an AuthenticationFilter resource that would be considered invalid.
+When an AuthenticationFilter is described as invalid, it could be for these reasons:
+
+- An AuthenticationFilter deployed with an empty `Realm` value
+- An AuthenticationFilter deployed with an empty `secretRef.Name` value
```

Comment:
I don't think we verify OpenAPI schema on our other policies, so these can be skipped.